### PR TITLE
Simplify navbar

### PR DESCRIPTION
**Pull request by palewire**
This patch aims to streamline the navbar code without any changes in design or functionality. It does the following:
- Breaks the CSS out into a new component
- Removes unused CSS from bootstrap's nav and collapse systems
- Refactors the HTML scaffolding for simplicity
- Introduces responsive "art direction" of which logo is displayed at different viewports
- Mixes in SCSS variables and other bits of sugar

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -42,44 +42,43 @@
...
-                           {{nav.title}}
-                       </a>
-                    </li>
-                    {%- endfor %}
-                </ul>
-            </div>
+    <nav class="navbar" role="navigation">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="/">
+               <picture>
```
Comment (choldgraf) [inline _layouts/default.html:48]: This is cool, I had no idea this existed in HTML

File: _layouts/default.html [hunk 2]
```diff
@@ -42,44 +42,43 @@
...
+        <button type="button"
+                class="hamburger"
+                data-bs-toggle="collapse"
+                data-bs-target="#bs-navbar-collapse" />
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar white-icon-bar"></span>
+            <span class="icon-bar white-icon-bar"></span>
+            <span class="icon-bar white-icon-bar"></span>
+        </button>
+        <div class="mobile-clearfix"></div>
```
Comment (choldgraf) [inline _layouts/default.html:68]: what does this do?

**Review by choldgraf — APPROVED** [review]
This looks great to me - I tried it out on widescreen + mobile, and neither seemed to look any different (which I think is what we want!)

A couple minor comments but this LGTM in general.

File: _layouts/default.html (hunk 1 above)
Comment (palewire) [reply _layouts/default.html:48]: There's a very nice article on it here https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

File: _layouts/default.html (hunk 2 above)
Comment (palewire) [reply _layouts/default.html:68]: The clearfix kicks the <ul> with the floating page links down below the logo block with a `clear:both`. There is almost certainly another way to do this. I was sticking with the same method used by the existing nav.

**Pull request merged**